public/app/plugins/datasource/loki/sortDataFrame.ts

```diff
@@ -1,6 +1,9 @@
 import { DataFrame, Field, SortedVector } from '@grafana/data';
 
-type SortDirection = 'ASCENDING' | 'DESCENDING';
+export enum SortDirection {
+  ascending,
+  descending,
+}
```

Comment:
```suggestion
export enum SortDirection {
  Ascending = 'ascending',
  Descending = 'descending',
}
```

I would change these to be capitalised and have them as string enums for better readability. But that is just a personal preference. I don't think we have anything about enums naming in our style guide.

Reply:
Good suggestion, changed in 9161fe7.